tests/end_to_end/test_op.py

```diff
@@ -97,3 +100,27 @@ def test_binops(execute):
     assert res.values["r10"] == 11
     assert res.values["r11"] == 9
     assert res.values["r12"] == 2
+
+
+def test_walrus(execute):
+    if sys.version_info >= (3, 8):
```

Comment:
you can look at `test_source_giver_adds_end_lineno` to see how to skip using pytest's inbuilt skip mark to flag the test appropriately as skipped rather than passed.

Reply:
Updated this: used a pytest skipif annotation for the function